common/src/main/java/com/genexus/CommonUtil.java

```diff
@@ -637,19 +637,37 @@ public static Date ymdhmsToT_noYL(int year, int month, int day , int hour , int
 						hour ++;
 					}
 				}
-				return nullDate();
+				hour = 0;
+				return tryNullDateOrTime(year, month, day, hour, minute, second, millisecond);
 			}
 			else
 			{
-				return nullDate();
+				return tryNullDateOrTime(year, month, day, hour, minute, second, millisecond);
 			}
 		}
 		catch (Exception e)
 		{
-			return nullDate();
+			return tryNullDateOrTime(year, month, day, hour, minute, second, millisecond);
 		}
 	}
 
+	private static Date tryNullDateOrTime(int year, int month, int day , int hour , int minute , int second, int millisecond)
+	{
+			if ((year ==1 && month == 1 && day ==1) || (hour == 0 && minute == 0 && second == 0 && millisecond == 0))
```

Comment:
I don't see clearly the meaning of the condition. 
Does it work ok for a DateTime like 1/1/1 11:10:10 for example?.